operations/configuration_management/Specifying_Hosts_and_Groups.md

```diff
@@ -0,0 +1,17 @@
+# Specifying Hosts and Groups
+
+When using the Configuration Framework Service (CFS), there are many steps where users may need to specify the hosts that CFS should configure. This can be done by specifying individual hosts, or groups of hosts. There are several places where a user may need to provide this information, particularly groups, and depending on where this information is provided, the behavior can change greatly.
+
+## Inventories
+
+CFS has multiple options for generating inventories, but regardless of which option is used, the information is then converted into an Ansible inventory/hosts file. The inventory is the list of components that Ansible can run against.  Anything not in the inventory is unknown to Ansible. Components in an inventory can be placed into groups so that they can be easily referenced together when specified by CFS.
```

Comment:
```suggestion
CFS has multiple options for generating inventories, but regardless of which option is used, the information is then converted into an Ansible inventory/hosts file. The inventory is the list of components that Ansible can run against. Anything not in the inventory is unknown to Ansible. Components in an inventory can be placed into groups so that they can be easily referenced together when specified by CFS.
```